### ISSUE TITLE
Unable to import and merge newest version of JWL backup files

### ISSUE THREAD

**Issue by mattndhatt** [AntonyCorbett/JWLMerge#94]
Unable to import and merge newest version of JWL backup files. 
- I go to import my files I get no activity in the program
- if I take an older file and add it in that one shows up right away
- however it still isn't letting me merge the older files now

Tried using a similar IOS program - had a similar issue, not working with the updated JWL files. 
- error report with that program was: "schemea version is incompatible. Should be 8 is 13. You might need to upgrade to a newer version of JW library first"
- hope this can help

Thanks so much for your work on this brother.

**Comment by AntonyCorbett** [comment]
@mattndhatt Note that this project is no longer compatible with the latest version of the JW Library. In July 2023 the database schema was changed from v8 to v13 and I will not be updating JWLMerge to support the new version. In Jan 2023, the JW Library licence agreement was strengthened, but it also gave opportunity for 3rd party developers to apply for official permission to do the sort of work that is present in JWLMerge. I made an application in January but have not received permission, and in these circumstances feel it best to archive the project at this juncture. Thank you for your understanding, and for your valuable input and feedback on the project over the last 6 years.
